### PR TITLE
Remove not required headers

### PR DESCRIPTION
**Pull request by kant2002**

### CODE REVIEW

File: src/fheroes2/maps/maps_fileinfo.cpp [hunk 1]
```diff
@@ -36,7 +36,6 @@
 #include "maps_fileinfo.h"
 #include "race.h"
 #include "settings.h"
-#include "world.h"
```
Comment (ihhub) [inline src/fheroes2/maps/maps_fileinfo.cpp:39]: Please add this code to fix compilation on MacOS:
```
#ifdef WITH_XML
#include "tinyxml.h"
#endif
```

**Review by ihhub — COMMENTED** [review]
Hi @kant2002 , I left a comment how to fix compilation on MacOS. Could you please take a look?

File: src/fheroes2/maps/maps_fileinfo.cpp (hunk 1 above)
Comment (kant2002) [reply src/fheroes2/maps/maps_fileinfo.cpp:39]: Didn't I push them? Sigh...

**Comment by ihhub** [comment]
@kant2002 , great thanks for these enhancements!

**Pull request merged**